### ISSUE TITLE
Allow cookies notice at the bottom seems to be cached

### ISSUE THREAD

**Issue by ADDISON74** [nexcess/magento-turpentine#1061]
This is a Magento 1.9.2.2 + Sample pack, no changes, just using Turpentine extension.

1) Go to Backend, System -> Configuration -> Web. In section "Session Cookie Management" set "Cookie Restriction Mode" option to Yes.

2) Flush the cache and reload the Frontend. You will get a notice at the very bottom of the page informing you "This website requires cookies to provide all of its features [...]". Click on [Allow] button. We have two behaviors and this is a Magento issue not a Turpentine one. In Chrome you will be redirected to noCookies page. In Firefox everything goes fine, the notice will disappear and you will get a new browser cookie named user_allowed_save_cookie having this value {"1":1}.

3) Let's go back in Backend to Web again this time in section "Browser Capabilities Detection". Set "Redirect to CMS-page if Cookies are Disabled" option from Yes to No. Flush the cache and reload the Frontend. 

- Chrome browser: clicking on [Allow] button for allowing cookies you will be redirect to noCookies page. Notice about cookies is still at the bottom. Setting up a new cookie named user_allowed_save_cookie with the above value it won't make the notice disappearing.

- Firefox browser: clicking on [Allow] button for allowing cookies page is reloaded and the notice is still there. Checking the browser cookies user_allowed_save_cookie has this value {"1":1} which is correct. 

Notice about allowing cookies is a cached block. For proving this I skipped Varnish by connecting Pound directly to Apache and flushed the cache and all sessions. 

- Chrome browser still has the same issue. Setting up that cookie will make the notice disappearing.

- Firefox browser: clicking on [Allow] button everything goes fine, the notice will disappear and you will get the same cookie user_allowed_save_cookie. 

Please in your next version of Turpentine include an ESI rule for this block and check if going back or forward in browser the notice is still hidden as it should be.

**Comment by aricwatson** [comment]
I just created a PR https://github.com/nexcess/magento-turpentine/pull/1065 that addresses this - please test and see if it works for you, it's working fine in my testing (so far!).

**Comment by ADDISON74** [comment]
I tested a few moments ago and the issue seems to be gone. Please continue testing and if it's working as expected do not forget to include in the devel branch and master.

**Comment by ADDISON74** [comment]
Check this out: turpentine_esi.xml file still has this code commented (I guess it is in master version too. I am using only devel version). Someone evaluated in the past the code, did not set up correctly the policy, and left it commented:

```php
<!--	<reference name="global_cookie_notice">
	    <action method="setEsiOptions">
		<params>
		    <access>public</access>
		</params>
	    </action>
	</reference>
-->
```

**Comment by aricwatson** [comment]
Hmm, I'm not seeing this in either *devel* or *master*...

**Comment by miguelbalparda** [comment]
This should be solved by #1065
